docs/concepts/hybrid-custody/guides/account-model.mdx

```diff
@@ -0,0 +1,334 @@
+
+Therefore, the presence of a `Manager` in an account implies there are potentially associated accounts for which the owning account has delegated access. This resource is intended to be configured with a pubic Capability enabling querying of an account's child account addresses via `getAccountAddresses()` and `getOwnedAccountAddresses()`. As you can deduce from these two methods, there is a notion of "owned" accounts which we'll expand on in a bit.
+
+A wallet or marketplace wishing to discover all of a user’s accounts and assets within them can do so by first looking to the user’s `Manager`.
+
+## Identifying Account Hierarchy
+
+To clarify, insofar as the standard is concerned, an account is a parent account if it contains a `Manager` resource, and an account is a child account if it contains at minimum a `ChildAccount` or additionally a `ProxyAccount` resource.
+
+We can see that the user’s `Manager.accounts` point to the address of its child account in its index, with corresponding Capabilities giving the Manager access to those accounts. Likewise, the child account’s `ProxyAccount.parentAddress` points to the user’s account as its parent address. This makes it easy to both identify whether an account is a parent, child, or both, and its associated parent/child account(s). `ChildAccount` resources underly all account delegations, so can have multiple parents whereas `ProxyAccount`s are 1:1. This provides more granular revocation as each parent account has its own Capability path on which its access relies.
+
+### Restricted vs. Owned Accounts
+
+It's worth noting here that `ProxyAccount` Capabilities enable access to the underlying account according to rules configured by the delegator. The `ProxyAccount` maintains these rules along with a `ChildAccount` resource within which the AuthAccount Capability is stored. Anyone with access to the surface level `ProxyAccount` can then access the underlying AuthAccount, but only according the the pre-defined rule set. These rules are fundamentally a list of Types that can/cannot be retrieved from an account.
+
+![HybridCustody Conceptual Overview](resources/hybrid_custody_conceptual_overview.png)
+
+So, if an app developer wants to enable Hybrid Custody but doesn't want to allow parent accounts to access FungibleToken Vaults, for example, the app developer can codify rule sets on allowable Capability types in a `CapabilityFilter` along with a `CapabilityFactory` defining retrieval patterns for those Capabilities. When delegation occurs, they would provide the `CapabilityFilter` and `CapabilityFactory` Capabilities to a `ChildAccount` which wraps the given filter & factory Capabilities in a `ProxyAccount` along with a Capability to itself before publishing the new `ProxyAccount` Capability for the specified parent account.
```

Comment:
This paragraph mentions not allowing access to Fungible Token Vaults, but then doesn't actually mention what a developer would specifically do to do that. I feel like we need to do the general description first, then expain the Vault use case. I took a stab, but feel free to change if it doesn't work well.

```suggestion
The app developer can codify these rule sets on allowable Capability types in a [`CapabilityFilter`](LINK) along with a [`CapabilityFactory`](LINK) defining retrieval patterns for those Capabilities. 
When delegation occurs, the developer would provide the `CapabilityFilter` and `CapabilityFactory` Capabilities to a `ChildAccount` resource which stores them in a `ProxyAccount` resource.  Then, capabilities are created for the `ChildAccount` and `ProxyAccount` resource and are given to the specified parent account.

So, if an app developer wants to enable Hybrid Custody but doesn't want to allow parent accounts to access FungibleToken Vaults, for example, they would specify in the `CapabilitiyFilter` that Vaults are not allowed to be accessed by the parent account, and that type restriction would therefore be enforced by the child account through its objects.
```

Reply:
Good point. Technically, a developer disallows access to FungibleToken Vaults by default since the accessible types must be enumerated in a Filter or provided in a Proxy. But it's worth highlighting that fact and the mechanisms that enable that enforcement